src/Apps/Collect/Components/Filters/FilterContainer.tsx

```diff
@@ -99,7 +99,11 @@ export class FilterContainer extends React.Component<
                 <Mobile filters={filters} />
               </Media>
               <Media greaterThan="xs">
-                <Desktop filters={filters} />
+                {(className, renderChildren) =>
+                  renderChildren && (
+                    <Desktop filters={filters} className={className} />
+                  )
+                }
```

Comment:
@alloy - can you explaiun why you would do this over just 

```js
<Media greaterThan="xs">
  <Desktop filters={filters} className={className} />
</Media>
```
?

Reply:
Values that are essential to determining the overal layout should either go _outside_ of `Media` usage or the wrapper div should be skipped entirely so that the 25% becomes relative again to the parent of `Media`. (I tried to explain in the commit message too https://github.com/artsy/reaction/pull/1534/commits/cde859bfa0389898118fd9b83552888b6b9713c8.)

There’s probably a better way to refactor this, but I didn’t want to get distracted and so went with the easy route that I knew would work.

Reply:
(Skipping the wrapper div is done by using a render prop)

Reply:
Here’s a better refactor example https://github.com/artsy/reaction/pull/1534/files#diff-e3efa0e5ac989dfe1a9073a3fc6c8384

Reply:
Put differently, just as with outer margins, a child component should not make overall layout decisions, instead the parent that composes the various components should be in charge.